webrtc.html

```diff
@@ -509,33 +509,47 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
+          </li>
+
+          <li>
+            <p>If <code>operations</code> is empty, execute the
+              function and let <var>p</var> be the promise returned by
+              the function.  If <code>operations</code> is non-empty,
+              let <var>p</var> be a promise equivalent to the promise
+              that would be returned by calling the function (it will
+              be settled at the same time and with the same result).
```

Comment:
How can I make two promises be "equivalent" and "settled at the same time" in JavaScript? That doesn't sound implementable. Do you mean one that settles *after* the other and with the same result?

Even then, how am I supposed to do that? The spec should tell me, not leave it up to me, or we wont have the same  deterministic timing behavior across browsers.

If you could show me the code you have in mind for this (akin to https://github.com/w3c/webrtc-pc/pull/307#issuecomment-142479604), then that would help me, and we could work backwards from that to figure out how to describe it in a way we can get agreement on.